src/ui/gradio_ui.py

```diff
@@ -49,7 +51,7 @@ def chat_ui(self, prompt, history, use_history=None):
                     f"Request failed with status code {response.status_code}"
                 )
                 self.logger.info(f"Response text: {response.text}")
-                return "Sorry, an error occurred: " + response.text
+                return f"Sorry, an error occurred: {response.text}"
```

Comment:
I don't particularly care either way, but is there a particular reason to prefer this style of string construction?

Reply:
well Python people find it more idiomatic. TBH I don't care either (returning to Python from Go world), but IMHO we should be consistent

Reply:
> IMHO we should be consistent

yeah, i'm also coming from a golang world so i miss the consistent structures the language itself enforces.

I was just curious if there was an actual performance difference or anything.  I'm fine w/ us trying to standardize on use of f-string.

Reply:
no, no performance improvements - Python is still slow, as usual :-)  [and Python 3.11/3.12 is a just bit better: some benchmarks I did https://www.root.cz/obrazek/1110447/]